arches/app/models/models.py

```diff
@@ -87,7 +87,9 @@ class DDataType(models.Model):
     datatype = models.TextField(primary_key=True)
     iconclass = models.TextField()
     defaultwidget = models.ForeignKey(db_column='defaultwidget', to='models.Widget')
-
+    config = JSONField(blank=True, null=True, db_column='config')
```

Comment:
See above note re: renaming `d_data_types.config` to `d_data_types.defaultconfig`